deployments/dev/environment.py

```diff
@@ -35,36 +35,35 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DEPLOYMENT_STAGE': 'dev',
 
         'AZUL_CATALOGS': json.dumps({
-            name: dict(atlas=atlas,
-                       internal=internal,
-                       plugins=dict(metadata=dict(name='hca'),
-                                    repository=dict(name='tdr')))
-            for name, atlas, internal in [
-                ('dcp2', 'hca', False),
-                ('dcp2ebi', 'hca', False),
-                ('lungmap', 'lungmap', False),
-                ('it2', 'hca', True),
-                ('it2ebi', 'hca', True),
-                ('it3lungmap', 'lungmap', True)
+            f'{name}{suffix}': dict(atlas=atlas,
+                                    internal=internal,
+                                    plugins=dict(metadata=dict(name='hca'),
+                                                 repository=dict(name='tdr')),
+                                    sources=sources)
+            for name, atlas, sources in [
+                (
+                    'dcp2',
+                    'hca',
+                    [
+                        'tdr:broad-jade-dev-data:snapshot/hca_dev_20201203___20210524_lattice:42',
+                        'tdr:broad-jade-dev-data:snapshot/hca_dev_20210621_managedaccess_4298b4de92f34cbbbbfe5bc11b8c2422__20210622:'
+                    ],
+                ),
+                (
+                    'dcp2ebi',
+                    'hca',
+                    ['tdr:broad-jade-dev-data:snapshot/hca_dev_20201023_ebiv4___20210302:4',]
+                ),
+                (
+                    'lungmap',
+                    'lungmap',
+                    ['tdr:broad-jade-dev-data:snapshot/lungmap_dev_20210412__20210414:',]
```

Comment:
Here and elsewhere, 

```suggestion
                    [
                        'tdr:broad-jade-dev-data:snapshot/lungmap_dev_20210412__20210414:',
                    ]
```

otherwise the trailing comma would not be needed.